### PR TITLE
RU: estimators.md

### PR DESCRIPTION
**Pull request by 0101011**

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks, Andrew.

Hi @nikolay-kha , can you review this when you get a chance? Thank you

File: site/ru/guide/estimators.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+    (Смотри подбробнее в статье [Загрузка данных и датасетов](../guide/datasets.md))
+
+2.  **Определение колонок параметров.** Каждая колонка `tf.feature_column`
+    определяет имя параметра, его тип и любую предварительную обработку
+    входных данных. Например, в следующем примере кода мы создадим три
+    колонки параметров, в которых будут хранится данные в формате целых
+    чисел или чисел с плавающей запятой. Первые две колонки параметров будут
+    просто идентифицировать имя и тип параметра. Третья колонка параметров указывает
+    лямбду-выражение, которое будут вызываться для оценки необработанных
```
Comment (Nikolay-Kha) [inline site/ru/guide/estimators.md]: not sure, but "лямбд**а**-выражение" or "указывает **на** лямбду-выражение" might be better

File: site/ru/guide/estimators.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    *   тензор, содержащий одну или более меток
+
+    Например, в коде ниже показан пример основного скелета для функции ввода
+    данных:
+
+        def input_fn(dataset):
+           ...  # манипулирует датасетом, извлекая словарь параметров и метки
+           return feature_dict, label
+
+    (Смотри подбробнее в статье [Загрузка данных и датасетов](../guide/datasets.md))
```
Comment (Nikolay-Kha) [inline site/ru/guide/estimators.md]: -> под**б**робнее
extra "б"

File: site/ru/guide/estimators.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    лямбду-выражение, которое будут вызываться для оценки необработанных
+    данных:
+							
+	# Определим три числовых колонки параметров.
+        population = tf.feature_column.numeric_column('population')
+        crime_rate = tf.feature_column.numeric_column('crime_rate')
+        median_education = tf.feature_column.numeric_column('median_education',
+                            normalizer_fn=lambda x: x - global_education_mean)
+	
+3.  **Укажем подходящий готовый Estimator.**  Например так мы укажем
```
Comment (Nikolay-Kha) [inline site/ru/guide/estimators.md]: comma after "например"

File: site/ru/guide/estimators.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    для поиска Estimator, который покажет лучшие результаты
+4.  Возможно потребуется улучшить нашу модель при помощи создания нашего
+    собственного Estimator.
+
+
+
+## Создание Estimators из моделей Keras 
+
+Ты можешь конвертировать уже имеющиеся у тебя модели Keras в Estimators. Это позволит
+тебе использовать все преимущества Estimators для данной модели, например для распределенного
```
Comment (Nikolay-Kha) [inline site/ru/guide/estimators.md]: comma after "например"

**Comment by lamberta** [comment]
@Nikolay-Kha Thank you for the review

@0101011 Please update when you get a chance and we can publish this one, too

**Comment by 0101011** [comment]
@lamberta This one is ready!

**Comment by 0101011** [comment]
@Nikolay-Kha thank you for your help 👍

**Pull request merged**